api/envoy/config/overload/v3/overload.proto

```diff
@@ -80,6 +82,31 @@ message Trigger {
   }
 }
 
+message ScaleTimersOverloadActionConfig {
+  enum TimerType {
+    // Adjusts the idle timer for downstream HTTP connections that takes effect when there are no active streams.
+    HTTP_DOWNSTREAM_CONNECTION_IDLE = 0;
```

Comment:
Is it worth mentioning this override in route_components.proto near:
google.protobuf.Duration idle_timeout = 24;

For discoverability.

Reply:
Also, add here a reference to the timeout parameter that is affected by this min override.